src/chainparams.cpp

```diff
@@ -218,15 +218,21 @@ class CMainParams : public CChainParams {
         nIssueSubAssetBurnAmount = 100 * COIN;
         nIssueUniqueAssetBurnAmount = 5 * COIN;
         nIssueMsgChannelAssetBurnAmount = 100 * COIN;
+        nIssueQualifierAssetBurnAmount = 1000 * COIN;
```

Comment:
obv need to settle on these values somehow...

Reply:
Yes, these are just placeholders until the actual values are decided.